ros_speech_recognition/README.md

```diff
@@ -96,14 +104,26 @@ This package uses Python package [SpeechRecognition](https://pypi.python.org/pyp
 
   Seconds of waiting for speech
 
+* `~audio_topic` (`String`, default: `audio`)
+
+  Topic name of input audio data
+  
 * `~depth` (`Int`, default: `16`)
 
   Depth of audio signal
   
+* `~channels` (`Int`, default: `1`)
```

Comment:
does this mean number of channel?
or id of channel?

Reply:
Thank you for your review!

This rosparam means the total number of channels in audio channels.
I modified the description.

Reply:
in that case, maybe it is better to use `n_channel` or something for param name.

Reply:
OK.

I replace `channels` param with `n_channels`.